BUILDING.md

```diff
@@ -210,10 +210,10 @@ buildscript {
 3. Add `apply plugin: 'opensearch.pluginzip'` to the build.gradle file.
 Once added, this should list the new task `publishPluginZipPublicationToZipStagingRepository`.
 
-4. Run the task `publishPluginZipPublicationToZipStagingRepository` (add it to managed build script build.sh)  
+4. Run the task `publishPluginZipPublicationToZipStagingRepository` (add it to managed build script `build.sh`)  
 ```./gradlew publishPluginZipPublicationToZipStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER```
 
-    Note: The gradle assemble task `./gradlew assemble` should be called first before calling `publishPluginZipPublicationToZipStagingRepository`, as zip file need to be generated first before publishing.
+    Note: The gradle assemble task `./gradlew assemble` should be called first before calling `publishPluginZipPublicationToZipStagingRepository`, as zip file need to be generated before publishing.
```

Comment:
Not sure if plugins support `./gradle assemble`, i know few plugins use `./gradle build`[1]

[1] https://github.com/opensearch-project/anomaly-detection/blob/main/DEVELOPER_GUIDE.md#building-from-the-command-line

Reply:
Thanks for the inputs @saratvemulapalli and @ylwu-amzn, the idea to mention the zip has be available (either by calling `assemble` or `build`) before publishing, let me rephrase this way and let me know this good to go with.

Note: The gradle task responsible to generate the distribution zip should be called first before executing `publishPluginZipPublicationToZipStagingRepository`, as the zip file needs to be generated before publishing.